### DISCUSSION TITLE
Welcome to ona Discussions!

### DISCUSSION THREAD

**Discussion by mattpascoe** [opennetadmin/ona#165] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by robfantini** [comment]
Hello Matt
 we are using version 18.1.1 .   
I have been using that tar fie method for install and upgrades..

I would like to change over to using  git .

 If the change over to git is possible,could you tell me the steps to do so?

thanks

Robert Fantini
